docs/semconv/database.md

```diff
@@ -261,6 +265,36 @@ This metric is [optional](https://github.com/open-telemetry/semantic-conventions
 |--------------------------|-----------------|------------------|----------------------------------------------------------------------------------------------|
 | `db.overflow.user.count` | UpDownCounter   | `{overflowUser}` | Number of times a User thread attempted to acquire a lock when no locks were available.      |
 
+### Metric: `db.cache.read.ratio`
+This metric is [optional](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#opt-in).
+
+| Name                  | Instrument Type   | Units (UCUM)       | Description                                                                              |
+|-----------------------|-------------------|--------------------|------------------------------------------------------------------------------------------|
+| `db.cache.read.ratio` | UpDownCounter     | `{cacheReadRatio}` | Percentage of page reads for this buffer pool that were satisfied by a cached page image |
```

Comment:
About the "Instrument Type" and "Units", like the description said, `db.cache.read.ratio` reflects the ratio of page writes in the buffer pool that are satisfied by a cached page image at a specific moment or over a period. This is a state rather than a cumulative change. Therefore, a **Gauge** is more appropriate for representing such an instantaneous ratio or percentage. At the same time, using the unit **1** is a standard and appropriate way to represent a ratio in OpenTelemetry.